perl/bin/cgpVaf.pl

```diff
@@ -1,7 +1,4 @@
-#!/software/perl-5.16.2/bin/perl -w
-
-eval 'exec /software/perl-5.16.2/bin/perl -w -S $0 ${1+"$@"}'
-    if 0; # not running under some shell
+#!/use/bin/perl
 
 ##########LICENCE############################################################
 # Copyright (c) 2016 Genome Research Ltd.
```

Comment:
You should update the date range on any file that you edit.  This should read 2016-2017